script/draft-release-notes

```diff
@@ -50,6 +50,31 @@ async function main() {
     let notes = commit.releaseNotes;
     if (commit.pr == "") {
       link = "https://github.com/zed-industries/zed/commits/" + commit.hash;
+    } else if (channel == "flatpak") {
+      const lines = notes.split("\n");
+      for (var i = 0; i < lines.length; i++) {
+        line = lines[i];
+        line = line.replaceAll("<", "&lt;");
+        line = line.replaceAll(">", "&gt;");
+        line = line.replace(/^\s*(-|\*)/, ""); // remove bullet point
+        line = line.replaceAll(/\(\[(#|\d|\s)+\]\((\w|:|\/|\.|-)*\)\)/g, ""); // remove links to issues
+        line = line.replaceAll(/\(#(\d|\s)*\)/g, ""); // remove issue numbers in brackets
+        line = line.replace(/\s+\./, ""); // remove spaces before periods
+        line = "<li>" + line.trim() + "</li>"
+        line = line.replace(/([^.])<\/li>$/, '$1.</li>'); // add missing periods
+
+        if (/#\d*/.test(line)) { // if the notes contain an issue number without brackets or a link, it likely is the low-effort type flathub wants to avoid
+          continue;
+        } else if (line.length > 200) {
```

Comment:
nit: This `else if` can just be a consecutive `if` since the previous condition terminates.